src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/PrimarySite/index.js

```diff
@@ -37,130 +37,138 @@ const StepPrimarySetup = () => {
 		setIsDrawerSuppressed( true );
 		setDrawerActiveView( VIEW_NAV_GET_STARTED );
 		setIsHeaderNavigationEnabled( true );
+		getSiteClassificationsData();
 	}, [] );
 
-	const [ clickedIndex, changeCategory ] = useState( -1 );
-	const [ inputCategVal, changeInputCateg ] = useState( '' );
+	const [ siteClassData, setSiteClassData ] = useState();
+	const [ primaryCategory, changePrimaryCategory ] = useState( '' );
+	const [ inputFieldValue, setInputFieldValue ] = useState( '' );
```

Comment:
```suggestion
	const [ custom, setCustom ] = useState( '' );
```
Do we need a separate state for handling changes to the input value? The only extra thing that I see we are doing is adding `labelPri` as `custom` which has nothing to do with this state? Can we store the value in `primaryCategory` to be consistent and populate the input field only when `custom` is true?